src/twisted/conch/ssh/userauth.py

```diff
@@ -306,6 +318,23 @@ def auth_publickey(self, packet: bytes) -> Deferred[_ConchPortalTuple]:
             )
         return result
 
+    # Follows the OpenSSH SK signature format specified at
+    # https://github.com/openssh/openssh-portable/blob/a4aa090a3d40dddb07d5ebebc501f6457541a501/PROTOCOL.u2f#L176
+    @staticmethod
+    def _wrapSecurityKeySignedData(signature, originalSignedData, application):
```

Comment:
To make it easier to read ... and also be more specific for the "signature"

```suggestion
    @staticmethod
    def _wrapSecurityKeySignedData(u2fSignature, originalSignedData, application):
        """
        See https://github.com/openssh/openssh-portable/blob/a4aa090a3d40dddb07d5ebebc501f6457541a501/PROTOCOL.u2f#L176
        
        In addition to the message to be signed, the U2F signature operation
        requires the key handle and a few additional parameters. The signature
        is signed over a blob that consists of:

	    byte[32]	SHA256(application)
	    byte		flags (including "user present", extensions present)
	    uint32		counter
	    byte[]		extensions
	    byte[32]	SHA256(message)
	    
	    The signature format used on the wire in SSH2_USERAUTH_REQUEST:

	    string		"sk-ecdsa-sha2-nistp256@openssh.com"
	    string		signature
	    byte		flags
	    uint32		counter
        """
```